lib/seahorn/BvOpSem2.cc

```diff
@@ -2386,11 +2386,37 @@ Bv2OpSemContext::Bv2OpSemContext(SymStore &values, ExprVector &side,
   setPathCond(o.getPathCond());
 }
 
+Expr Bv2OpSemContext::convertConstArrayToLambda(Expr e) {
```

Comment:
we use the term `coerce` to express any form of coercion. Would be good to be consistent.

Reply:
done